test.ttl

```diff
@@ -0,0 +1,40 @@
+@prefix prov: <http://www.w3.org/ns/prov#> .
```

Comment:
Remove or move to `tests/static/...` if being used in a test

Reply:
not used

Reply:
Remove this file